berkeley-mobile/Common/ResourceTableViewCell.swift

```diff
@@ -8,6 +8,7 @@
 
 import UIKit
 
+// TODO: when backend gets images for resources, update this to conform to ImageViewCell
```

Comment:
We might want to do this now so that the app will be ready-to-go to present images for resources when the backend is updated. Don't worry about it though if this is too much work.